ipa.go

```diff
@@ -60,3 +60,9 @@ func FromBytes(fr *Fr, data []byte) {
 	copy(aligned[32-len(data):], data)
 	fr.SetBytes(aligned[:])
 }
+
+func MapToScalarFieldBytes(point *Point) [32]byte {
```

Comment:
Bikeshedding: HashPointToBytes/PointToHashBytes

Reply:
Would also add the LE postfix

Reply:
Considering we return `[32]byte` and not `Fr`, then sounds more like `HashPointToBytes` or similar is a better name than referring to "ScalarField" which seems sounds leaking details. Also agree on LE.